v1.1/uuid.md

```diff
@@ -0,0 +1,118 @@
+---
+title: UUID
+summary: The UUID data type stores 128-bit Universal Unique Identifiers.
+toc: false
+---
+
+<span class="version-tag">New in v1.1:</span> The `UUID` (Universally Unique Identifier) [data type](data-types.html) stores a 128-bit value that is [unique across both space and time](https://www.ietf.org/rfc/rfc4122.txt). 
+
+<div id="toc"></div>
+
+## Syntax
+A `UUID` value can be expressed using the following formats:
+
+Format | Description 
+-------|-------------
+Standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format | Hyphen-seperated groups of 8, 4, 4, 4, 12 hexadecimal digits.<br><br> Example: `acde070d-8c4c-4f0d-9d8a-162843c10333`
+With braces | The standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format with braces.<br><br>Example: `{acde070d-8c4c-4f0d-9d8a-162843c10333}`
+As `BYTES` | `UUID` value specified as bytes.<br><br>Example: `b'kafef00ddeadbeed'`
+`UUID` used as a URN | `UUID` can be used as a Uniform Resource Name (URN). In that case, the format is [specified](https://www.ietf.org/rfc/rfc2141.txt) as "urn:uuid:" followed by standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format.<br><br>Example: `urn:uuid:63616665-6630-3064-6465-616462656564`
+
+## Size
+A `UUID` value is 128 bits in width, but the total storage size is likely to be larger due to CockroachDB metadata. 
+
+## Examples
+
+### Create a table with manually-entered `UUID` values
+
+#### Create a table with `UUID` in standard [RCF4122](http://www.ietf.org/rfc/rfc4122.txt)-specified format.
```

Comment:
@Amruta-Ranade, let's remove the period from the end of these example names. Sorry, didn't see this earlier.